Classes/Data class copy/task-info.yaml

```diff
@@ -0,0 +1,10 @@
+type: edu
+files:
+- name: src/Task.kt
+  visible: true
+  placeholders:
+  - offset: 105
+    length: 33
+    placeholder_text: TODO()
+- name: test/Tests.kt
+  visible: true
```

Comment:
Is the test intentionally set to be visible?

Reply:
no